woocommerce/woocommerce-ios
# Zendesk: Migrate from 3.x to 4.x

Closes #1591 

This PR updates the Zendesk SDK from 3.x to 4.x.

### Changes
Migrating from 3.x to 4.x had some breaking changes to the code. Other than that, there are no major changes to the way we are currently using the SDK.

### To test
1. Open a support ticket using a non-registered agent email address.
2. After sending the ticket, use your registered agent email address to log in to Zendesk and check that the message was successfully received. Skim the attached app logs and meta data. 
3. Send a reply 
4. Check the reply inside the app

### Known issues
- The "My Tickets" nav bar title appears in black instead of white. A ZD support request has been opened about this issue. Related bug on our repo to track the issue: #1513.

Update release notes:
